_data/mentors.yml

```diff
@@ -2262,3 +2262,155 @@
     extra: Potential Mentoring Topics such as Software Development Strategies such as Best practices for planning, executing, and managing software projects effectively. .NET Development such as Deep dive into C#, ASP.NET MVC, and building robust applications using the .NET framework. ReactJS such as Developing dynamic and responsive web applications with ReactJS.
   network:
     - linkedin: https://www.linkedin.com/in/s-b-b51a76317/
+
+- name: Andrew  King
+  disabled: false
+  matched: false
+  sort: 200
+  num_mentee: 0
+  hours: 4
+  type: long-term
+  index: 67
+  location: Yorkshire, UK
+  position: DevOps Engineer, DRW
+  bio: |
+    Over 30 years in commercial IT, 40 years of coding experience, 20 years in financial firms. Know how to manage, manage up, team building, finding team fit. Many years of experience interviewing (on both side of the table).
+  image: |
+    assets/images/mentors/andrew__king.jpeg # TODO: Run download_image script to actually download the image
+  languages: English
+  availability: []
+  skills:
+    experience: 16+ Years
+    years: 16
+    mentee: |
+      Someone who wants to engage in meaningful and useful conversations
+    areas:
+    - Site Reliability Engineering
+    - Engineering management
+    - Distributed systems
+    - Security
+    - Data Science
+    languages: C, Go, Python, JavaScript
+    focus:
+    - Grow from mid-level to senior-level
+    - Switch from IC to a management position
+    - Change specialisation within IT
+    - Grow beyond senior level
+    - Grow from beginner to mid-level
+    extra: |
+      Systems design for reliability and risk minimisation. SDLC in practice with CI/CD in production. Interview techniques. When to fix, vs. when to seek a change.
+  network:
+  - linkedin: https://www.linkedin.com/in/andrewkingit/
+
+- name: Fernanda Lancao Morais Martins
```

Comment:
I think this mentor is there, but with the different name. So maybe it is better to update her name in the yml.